docs/metrics.md

```diff
@@ -27,16 +27,22 @@ HTTP code.
 ## Routes
 
 Routes are organized in the following fashion:
-`/_/metrics/<extension-type>/<location-name>/<metric-type>`
+`/_/metrics/<extension-type>/<site-name>/<metric-type>/<bucket>/<key>/<version-id>`
```

Comment:
I recommend keeping `location-name` as we are trying to be consistent with our wording.

Reply:
Also maybe enclose in `[` and `]` the set of optional bucket, key and version-id fields which depend on the route.

Reply:
Good call, this also needs to be updated to take `version-id` as a query param, reflecting recent changes.

Reply:
Updated accordingly.